pom.xml

```diff
@@ -214,6 +214,27 @@
       </resource>
     </resources>
     <plugins>
+      <!-- Do not deploy the root POM -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-deploy</id>
```

Comment:
Do we have to duplicate the id, phase and goals section?

Reply:
I opted to skip only the default execution (mvn deploy) instead of all executions.

It could just be:
```
        <configuration>
          <skip>true</skip>
        </configuration>
```